### PR TITLE
fix: add missing step in setup

### PR DESCRIPTION
**Pull request by BenediktEngel**
Added the missing step describing the div which is needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,7 +104,14 @@ viewer.addSimpleImage({
 ```
 > If no opacity is given the layer will have the default OSD opacity of 100% (or 1).
 
-3. Setup the additional information of the layers:
+3. Add DIV for Sliders
```
Comment (WasMachenSachen) [inline README.md:107]: 3. Add `div` for Sliders
> Because the plugin will position it `absolute`, it doesn't matter where you put it in the dom.

**Review by WasMachenSachen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BenediktEngel) [reply README.md:107]: Added via --amend

Comment (WasMachenSachen) [reply README.md:107]: you sure? cant see it.

Comment (WasMachenSachen) [reply README.md:107]: nevermind

**Review by WasMachenSachen — APPROVED** [review]

**Review by SickxX — APPROVED** [review]
ok.

**Pull request merged**